algorithms/linfa-ensemble/Cargo.toml

```diff
@@ -30,7 +30,10 @@ linfa-datasets = { version = "0.7.0", path = "../../datasets/", features = ["iri
 ndarray = { version = "0.15" , features = ["rayon", "approx"]}
 ndarray-rand = "0.14"
 rand = { version = "0.8", features = ["small_rng"] }
+approx = {version = "0.5"}
```

Comment:
do we need this module?

Reply:
Yes, in case of regressor we need to have a tolerance to approximate the values to compare if they are equal.